languages.toml

```
@@ -2713,3 +2713,15 @@ injection-regex = "wren"
 file-types = ["wren"]
 roots = []
 indent = { tab-width = 2, unit = "  "}
+
+[[language]]
+name = "todotxt"
+scope = "text.todotxt"
+file-types = ["todo.txt", "todotxt"]
```

```suggestion
file-types = [".todo.txt"]
```

This should work to catch `.todo.txt` files

---

Oh I see, I was thinking that `.todo.txt` was the full filename

---

You could use `suffix` here, I think that should catch `test.todo.txt`:

```toml
file-types = [{ suffix = ".todo.txt"}, "todotxt"]
```